static/app/api.tsx

```diff
@@ -581,3 +585,60 @@ export class Client {
     );
   }
 }
+
+function generateOrganizationBaseUrl(organizationUrl: string) {
+  return `${organizationUrl}${DEFAULT_BASE_URL}`;
+}
+
+type APIRouteType = 'organization-events';
+
+type LegacyRoute = string;
+type CustomerDomainRoute = string;
+type RouteTuple = [LegacyRoute, CustomerDomainRoute];
+
+const routeRenderMap: Record<APIRouteType, RouteTuple> = {
+  'organization-events': ['/organizations/:org_slug/events/', '/events/'],
```

Comment:
I wonder if we should look into ways to generate this mapping from the server defined URLs. It would be nice to not need to update this by hand. We can come back to that later when it becomes a problem though.